### ISSUE TITLE
Ensures elements with an ARIA role that require child roles contain them (aria-required-children ) in CommandBar

### ISSUE THREAD

**Issue by belozerovany** [microsoft/fluentui#15445]
### Environment Information

Package version(s): (7.0.1))
Browser and OS versions: (all)

### Describe the issue:
When we testing with [Accessibility Insights for Web](https://accessibilityinsights.io/docs/en/web/overview/) in CommandBar component find Accessibility error with Ensures elements with an ARIA role that require child roles contain them ([aria-required-children -](https://dequeuniversity.com/rules/axe/4.0/aria-required-children?application=msftAI))
 Please check https://prnt.sc/uw6x1l
Issue: Ensures elements with an ARIA role that require child roles contain them (aria-required-children)
How to fix:
Fix any of the following:
 Required ARIA children role not present: menuitemradio, menuitem, menuitemcheckbox
You may reproduce this error in mode "debug":
https://cdpn.io/belozerovany/debug/VwjYxWE/LQAExyBaxggr

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/belozerovany/pen/VwjYxWE?editable=true%3Dhttps%3A%2F%2Ffabricweb.z5.web.core.windows.net%2F
This default component from the UI Fabric

#### Actual behavior:
Ensures elements with an ARIA role that require child roles contain them (aria-required-children)

#### Expected behavior:
After checking with Accessibility Insights for Web we should not get an Accessibility errors

### Documentation describing expected behavior

Are you willing to submit a PR to fix? No
Requested priority: High
Products/sites affected: Products

**Comment by paulgildea** [comment]
@belozerovany Thanks for the issue.

@smhigley Is this related to the on-going changes in standard/tool?

**Comment by cnamal-msft** [comment]
@paulgildea Accessibility insights was [recently](https://github.com/microsoft/accessibility-insights-web/releases/tag/web%402.22.0) update and ships with a new version of [axe-core](https://github.com/dequelabs/axe-core/releases/tag/v4.0.2) which has:
>required-parent: fail if intermediate role is not the required parent

The command bar has the following HTML layout:

```html
<div role="menubar">
  <div role="group"> <!-- from OverflowSet -->
      <div role="menuitem">
      </div>
  </div>
</div>
````

Hope this helps

**Comment by paulgildea** [comment]
@JustSlone Do we have a main issue tracking this AI rule change?

**Comment by JustSlone** [comment]
@cnamal-msft yeah, seems the change to make this invalid was unintentional: https://github.com/dequelabs/axe-core/issues/2535

We worked around it in the meantime anyway though: https://github.com/microsoft/fluentui/pull/15511

closing this issue
